Specs/main.js

```diff
@@ -0,0 +1,158 @@
+var userInput, guess, randomNum;
+
+function randomNumber() {
+  randomNum = (Math.floor(Math.random() * 100) +1)
+  console.log(randomNum)
+}
+
+function guessTheNum() {
+  userInput = prompt("What's your guess of the number?", "")
+  guessNum = parseInt(userInput)
+  if(guessNum === randomNum) {
+    alert("you won!")
+  } else if(guessNum < randomNum) {
+    alert("your guess is too small")
+  } else if(guessNum > randomNum) {
+    alert("your guess is too high")
+  }
+
+  if(userInput === "exit") {
+    return null
+  }
+}
+
+var splitWords, checkIndex, num;
+
+function stringIndices() {
+  let wordsToParse = prompt("Enter words to parse", "")
```

Comment:
From what I've learned, you should name these const if you aren't calling them within your code to change the values